spyder/plugins/explorer/confpage.py

```diff
@@ -25,17 +27,40 @@ def setup_page(self):
 
         # Widgets
         general_widget = QWidget()
-        check_show_all = newcb(_("Show all files"), 'show_all')
+
+        # General options group
+        basic_group = QGroupBox(_("General options"))
         check_show_hidden_files = newcb(_("Show hidden files"), 'show_hidden')
         check_single_click = newcb(_("Single click to open files"),
                                    'single_click_to_open')
-        edit_filename_filters = self.create_textedit(
-            _("Show files with these extensions:"),
+        basic_layout = QVBoxLayout()
+        basic_layout.addWidget(check_show_hidden_files)
+        basic_layout.addWidget(check_single_click)
+        basic_group.setLayout(basic_layout)
+
+        # Filter options group
+        filter_group = QGroupBox(_("Filter settings"))
+        description_label = QLabel(
+            _('Filter files by name, extension, or more using '
+              '<a href="https://docs.python.org/3/library/glob.html">glob '
```

Comment:
```suggestion
              '<a href="https://en.wikipedia.org/wiki/Glob_(programming)">glob '
```

Reply:
I'm not sure how I feel about pointing to wikipedia instead of python docs. I would prefer leaving the python docs, what do you think @ccordoba12 and @steff456?

Reply:
I naturally would prefer the Python docs if practicable (I believe I originally suggested this link), but it doesn't actually explain what a glob is or how to write one, nor does it even include a link to another resource that does so. Almost everything there is about the glob module API (which isn't relevant here), and even the first paragraph is packed with highly technical jargon that puts what I've written to shame. This would be very confusing and intimidating for a newer user who just wants to know what the heck a glob is and how they write one (which is our purpose for including the link in the first place), and so I favor a much friendlier introduction instead (doesn't have to be a Wikipedia article, but something other than the highly technical API docs).

Perhaps @isabela-pf might have valuable input as well?

Reply:
I guess that the link that best describes to the user what a glob pattern is and how to create one should be the one we reference to, so I'll say Wikipedia is this case is better.

Reply:
I agree with @CAM-Gerlach and @steff456. We should use the Wikipedia link because it really describes how to use glob patterns.

Reply:
Okay if you all agree I’m fine with Wikipedia then 👍🏻